doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multidb
+=======
+
+To support computationally intensive tasks like scientific simulations,
+SmartSim provides functionality to automate the deployment of multiple
+databases on an HPC cluster. As data volume increases, consider augmenting
```

Comment:
This is a bit unclear as it doesn't quite capture the use cases that we've anticipated that people might need to use multiple databases. I think in the design doc, Bill and I listed a few use cases